pkg/serviceaccount/claims.go

```diff
@@ -176,23 +178,25 @@ func (v *validator) Validate(ctx context.Context, _ string, public *jwt.Claims,
 	podref := private.Kubernetes.Pod
 	noderef := private.Kubernetes.Node
 	secref := private.Kubernetes.Secret
-	// Make sure service account still exists (name and UID)
-	serviceAccount, err := v.getter.GetServiceAccount(namespace, saref.Name)
-	if err != nil {
-		klog.V(4).Infof("Could not retrieve service account %s/%s: %v", namespace, saref.Name, err)
-		return nil, err
-	}
+	if v.lookup {
```

Comment:
![image](https://github.com/user-attachments/assets/cd0d7d9f-370b-4e32-88fe-7f982d097af0)
This change is more straightforward than what GitHub is trying to tell you.